docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Did you happen to see this? https://daler.github.io/sphinxdoc-test/includeme.html

Reply:
readthedocs will provide an alternative to gh-pages. We can revisit if needed.